.github/ISSUE_TEMPLATE/000-crash-report.yml

```diff
@@ -0,0 +1,85 @@
+name: Crash Report
+description: Report a bug that crashes the game or prevents startup.
+labels: [ "status: unverified", "bug: crash" ]
+body:
+  - type: input
+    id: version
+    attributes:
+      label: GregTech CEu Version
+      description: The version of Gregtech CEu you were using when this bug was encountered. If you do not know where to find this, look for the mod jar file in the mods folder, or in the mods viewer in-game.
+      placeholder: "Example: v2.3.4"
+    validations:
+      required: true
+  - type: input
+    id: modpack
+    attributes:
+      label: Modpack Used
+      description: The modpack and its version used when this bug was encountered. If you were not using a modpack, please leave this field blank.
+      placeholder: "Example: GregTech Community Modpack v1.2.1"
+    validations:
+      required: false
+  - type: textarea
+    id: addons
+    attributes:
+      label: Addons Installed
+      description: The addons and their respective versions used when this bug was encountered. If you were not using addons, or do not know which are present (i.e. in a modpack), please leave this field blank.
+      placeholder: "Example: Gregicality Multiblocks v1.1.3"
+    validations:
+      required: false
+  - type: input
+    id: environment
+    attributes:
+      label: Environment
+      description: "How you were playing on the world. Typical answers include: Singleplayer, Open to LAN, Forge Server, Sponge Server, or Mohist Server."
+      placeholder: "Example: Singleplayer"
+    validations:
+      required: true
+  - type: dropdown
+    id: crash
+    attributes:
+      label: Side Crashed
+      description: Where the game crashed.
+      options:
+        - "Client"
+        - "Server"
+        - "Client and Server"
+    validations:
+      required: true
+  - type: dropdown
+    id: world
+    attributes:
+      label: New Worlds
+      description: Does this crash occur on a newly generated world?
+      options:
+        - "Yes"
+        - "No"
+        - "Unsure"
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Steps to Reproduce
+      description: How to reproduce the crash.
+      placeholder: "Example: 1) I did X..."
+    validations:
+      required: true
+  - type: textarea
+    id: crash-report
+    attributes:
+      label: Crash Report
+      description: "You can find the crash report in `installation/crash-reports/`, if one was created. Additionally, please attach `installation/logs/latest.log`. If either or both files do not exist, state such. You can add these files by dragging them into the text box below, or linking to a paste site with their content. **Do not paste the contents of either these files directly into the text box.**"
```

Comment:
Personally I would rather recommend gists and pastebin entirely over just attaching the crash for us to download, as I don't like downloading the logs every time.